rest/endpoints/ceph.py

```diff
@@ -0,0 +1,66 @@
+import json
```

Comment:
keep file name specific to operation `RBD_pools.py`

Reply:
no, the point is to keep it the way ceph-ci is written. Like I said, I am giving demo of framework design choices and tradeoffs, kindly join if interested.